curriculum/challenges/english/25-front-end-development/workshop-binary-search/683ec95f2f8781355556ff83.md

```diff
@@ -0,0 +1,56 @@
+---
+id: 683ec95f2f8781355556ff83
+title: Step 11
+challengeType: 20
+dashedName: step-11
+---
+
+# --description--
+
+You can see that the first call returns `[3]`. That's because `3` is the middle value in `[1, 2, 3, 4, 5]`.
+
+On the other hand, the second call returns `'This shows because the value is farther than the middle value to the right or left'`. This happens because, after checking the middle value, the loop currently lacks the logic to narrow the search range by updating the `low` or `high` variables. It only acts as a single midpoint check and then finishes without further searching.
+
+To allow the binary search to continue narrowing its search, add an `elif` block that checks if `value` is greater than `value_at_middle`. Add the `pass` keyword inside the `elif` for now.
+
+# --hints--
+
+You should add an `elif` with the condition `value > value_at_middle`. Don't forget to add the `pass` keyword into the `elif` block.
+
+```js
+({
+    test: () =>
+        assert(
+            runPython(
+                `_Node(_code).find_function("binary_search").find_whiles()[0].find_ifs()[0].is_equivalent("if value == value_at_middle:\\n    return path_to_target\\nelif value > value_at_middle:\\n    pass")`
+            )
+        )
+})
+```
```

Comment:
Allowing both solutions and not checking `pass`.
````suggestion
You should add an `elif` block for when `value` is greater than `value_at_middle`.

```js
({ test: () => runPython(`
_cond = _Node(_code).find_function("binary_search").find_whiles()[0].find_ifs()[0].find_conditions()[1]
assert _cond.is_equivalent("value > value_at_middle") or _cond.is_equivalent("value_at_middle < value")
`) })
```
````